common/src/main/java/com/microsoft/identity/common/internal/broker/BrokerTokenProvider.java

```diff
@@ -56,15 +55,15 @@ private Bundle getBrokerOptions(final MicrosoftAuthorizationRequest request, fin
         brokerOptions.putString(AuthenticationConstants.Broker.ACCOUNT_USERINFO_USERID, request.getLoginHint());
         brokerOptions.putString(AuthenticationConstants.Broker.ACCOUNT_LOGIN_HINT, request.getLoginHint());
         brokerOptions.putString(AuthenticationConstants.Broker.ACCOUNT_NAME, request.getLoginHint());
-        brokerOptions.putString(AuthenticationConstants.Broker.ACCOUNT_EXTRA_QUERY_PARAM, request.getExtraQueryParam());
+        // TODO uncomment the line below when broker support is added...
+        //brokerOptions.putString(AuthenticationConstants.Broker.ACCOUNT_EXTRA_QUERY_PARAM, request.getExtraQueryParam());
```

Comment:
When broker support is introduced to MSAL, this must be uncommented and routed through...

Leaving as TODO for now